lighthouse-core/gather/gatherers/dobetterweb/optimized-images.js

```diff
@@ -82,9 +82,11 @@ class OptimizedImages extends Gatherer {
       const isSameOrigin = URL.hostsMatch(pageUrl, record._url);
```

Comment:
just checked and confirmed an http page requesting an image from the same host but over https does indeed taint the canvas